crowdsec-docs/unversioned/console/blocklists/integrations/firewall.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Firewall integrations
+---
+
+The Firewall integration feature allows users to consume blocklists without installing any CrowdSec Security Engine. This feature provides flexibility by enabling users to pull blocklists from an endpoint in various formats.
+
+## Create an integration
+
+You will find the integration page under the blocklist menu. When it's your first time, you will encounter this page showing you all the possible integrations available.
+
+![](/img/console/blocklists/integrations/catalog.png)
+
+You can click on the desired provider or the generic vendor format to create a new integration.
```

Comment:
specify a bit that the generic vendor format is just "one ip per line" and is suitable if you don't know what format to use.